astropy/nddata/tests/test_ccddata.py

```diff
@@ -95,10 +58,10 @@ def test_ccddata_meta_header_conflict():
         assert "can't have both header and meta." in str(exc)
 
 
-@pytest.mark.data_size(10)
-def test_ccddata_simple(ccd_data):
-    assert ccd_data.shape == (10, 10)
-    assert ccd_data.size == 100
+def test_ccddata_simple():
+    ccd_data = create_ccd_data()
+    assert ccd_data.shape == (100, 100)
```

Comment:
I cannot comment on whether forcing everything 100x100 now is good or bad for the tests here, so I'll wait for @mwcraig and @crawfordsm to chime in.

Reply:
I'm not sure if it really makes sense too. The test here is actually was just testing the fixture (now testing the helper function). 😰 

However there were only two tests which used a different size - and just to check if it really had that size afterwards.

In the end I prefer tests to be completely self-contained **and** don't use `random` at all. However it's always hard (or at least time-consuming) to change this retrospectively because I would need to investigate which tests really need what. That's a bit too much effort for now.